### PR TITLE
create API to get user profile details

### PR DESCRIPTION
**Pull request by umiatiyah**
- create method to get list social media by user_profile_id
- create method to get list hobby by user_profile_id
- create method to get list experience by user_profile_id
- create method to get list education by user_profile_id
- create method to get list contact by user_profile_id
response from API userprofile/:userId:
res

### CODE REVIEW

File: response/user_profile.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package response
+
+type UserProfileResponse struct {
+	UserProfile interface{} `json:"user_profile"`
```
Comment (fathrezza146) [inline response/user_profile.go]: jgn pake innnterface. trus haruse ada db di dalam property haruse ketambah DB

**Review by fathrezza146 — CHANGES_REQUESTED** [review]

File: response/user_profile.go (hunk 1 above)
Comment (umiatiyah) [reply response/user_profile.go]: piye? bisa ga pake struct dari model yang udaah ada?

Comment (fathrezza146) [reply response/user_profile.go]: Bisa. Tinggal `json:"smth",db:"smth"`

**Review by fathrezza146 — APPROVED** [review]
LGTM!!!!!!

**Pull request merged**